index.bs

```diff
@@ -2789,6 +2789,31 @@ the [=authenticator=] MUST:
     attStmtTemplate .within $$attStmtType
     ```
 
+### Signature Formats for Packed Attestation, FIDO U2F Attestation, and Assertion Signatures ### {#signature-attestation-types}
+    - For COSEAlgorithmIdentifier -7 (ES256),  and other ECDSA-based algorithms,
+        a signature is encoded as an ASN.1 DER Ecdsa-Sig-Value, as defined in [[RFC3279]] section 3.3.1.
+
+        Note: As CTAP1/U2F devices are already producing signatures in this format, CTAP2
+        devices will also produce signatures in same format, for consistency reasons.
+        It is recommended that any new attestation formats defined not use ASN.1 encodings,
+        but instead represent signatures as equivalent fixed-length byte arrays without internal structure,
+        using the same representations as used by COSE signatures as defined in [[!RFC8152]].
+
+        ```
+        Example:
+        30 44                               ; SEQUENCE (68 Bytes)
+            02 20                            ; INTEGER (32 Bytes)
+            |  3d 46 28 7b 8c 6e 8c 8c  26 1c 1b 88 f2 73 b0 9a
+            |  32 a6 cf 28 09 fd 6e 30  d5 a7 9f 26 37 00 8f 54
+            02 20                            ; INTEGER (32 Bytes)
+            |  4e 72 23 6e a3 90 a9 a1  7b cf 5f 7a 09 d6 3a b2
+            |  17 6c 92 bb 8e 36 c0 41  98 a2 7b 90 9b 6e 8f 13
+        ```
+    - For COSEAlgorithmIdentifier -257 (RS256), `sig` contains the signature generated using the
```

Comment:
AFAICT the sig values produced by [[RFC8017]] sections 8.1.1 and 8.2.1 [are not ASN.1-wrapped](https://github.com/w3c/webauthn/issues/799#issuecomment-366102357).  If that is correct, would a Note: to that effect be helpful to readers here?

Reply:
Sure.